src/main/resources/assets/mekanism/lang/en_us.lang

```diff
@@ -805,6 +805,7 @@ tooltip.flamethrower.modeBump=Flamethrower mode bumped to
 
 tooltip.seismicReader.needsEnergy=Not enough energy to interpret vibration
 tooltip.seismicReader.noVibrations=Unable to discover any vibrations
+tooltip.SeismicReader=A portable machine that uses seismic vibrations to provide information on differing layers of the world.
```

Comment:
English is fine, but while we're at it, shouldn't we make it `tooltip.mekanism.SeismicReader`?

Reply:
I've got a separate branch which will update all translations at once ;) Will update that later today

Reply:
I know, but would it not be easier to add the new one with the new schema already?

Reply:
Just looked, I apparently have updated tooltips already.. Will change this here ;)